### DISCUSSION TITLE
Presidio for personal use with scanning for PII on OneDrive?

### DISCUSSION THREAD

**Discussion by LayersOfAbstraction** [microsoft/presidio#1483] [Requested features]
Hello,

While I am aware this can be packaged in a container, I am wondering if presidio is suitable for home use to scan a OneDrive client for PII on Windows 11 Pro? If not should I connect to OneDrive with rclone and scan through that connection? Should I be running it on WSL? Just looking to ensure I get something that fits home needs.

**Comment by omri374** [comment]
A few approaches that could work:
1. Leverage [Presidio-CLI](https://github.com/microsoft/presidio/tree/main/presidio-cli) which can scan folders and return the found PII
2. Use the [Presidio - Azure Data Factory sample](https://github.com/microsoft/presidio/blob/main/docs/samples/deployments/data-factory/presidio-data-factory-template-gallery-http.md), by connecting Presidio to ADF and leverage the connector to M365: https://learn.microsoft.com/en-us/azure/data-factory/load-office-365-data
3. Write a simple Python script which scans the local OneDrive folder and stores where PII was found
